tracer/src/Datadog.Trace/Telemetry/DTOs/ConfigTelemetryData.cs

```diff
@@ -14,6 +14,7 @@ internal static class ConfigTelemetryData
         public const string AgentUrl = "agent_url";
         public const string AgentTraceTransport = "agent_transport";
         public const string Debug = "debug";
+        public const string IsManagedOnly = "managed_only";
```

Comment:
My level of care is low, but "instrumentation_attached" or something about the instrumentation might make the key a little more meaningful

Reply:
👍 In my "alternatives" I suggest ways of grabbing the version number of the auto-instrumented part. If that's the case, then we could potentially change this flag to be `NativeVersion` which would be `None` (or similar) if not available?